Racing/Game.cs

```diff
@@ -0,0 +1,232 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Racing
+{
+    public static class Game
+    {
+        public static object consoleLocker = new object();
+
+        public static Score score = new Score();
+        public static Lives lives = new Lives(Config.LivesNumber);
+        public static Motion motion = new Motion();
+
+        public static RoadSide roadSide = new RoadSide();
+        public static RivalCar rivalCar = new RivalCar();
+        public static RivalCar rivalCar2 = new RivalCar(Config.RivalCarOffset);
+        public static PlayerCar playerCar = new PlayerCar();
+
+        public static Key key = new Key();
+
+        public static Thread keyboardThred = new Thread(key.StartReadKey);
+
+        public static int roadSpeedK = 0;
+        public static int rivalSpeedK = 0;
+        public static int carCounter = 0;
+        public const int carCounterDefault = 2;
+        public static int carCounterK = carCounterDefault;
+        public static int MoveTimeout;
+        public static bool IsPaused = false;
+
+
+        //
+        // Initialize all game modules.
+        //
+        public static void Initialization()
+        {
+            key.keyEvent += KeyboardHandler;
```

Comment:
Do you remember that for instance member we use 'this.' statement and for static members we use 'Class name' instead.

Reply:
-1 to Source code quality